studio/Python/tinymovr/iface/can_bus/endpoints.py

```diff
@@ -386,4 +386,14 @@
         "defaults": {"timeout": 3},
         "labels": ("enabled", "timeout")
     },
+    "set_vel_inc":
+    {
+        "description": "Set vel inc.",
+        "type": "w",
+        "ep_id": 0x02B,
+        "types": (DataType.FLOAT,),
+        "units": ("ticks/second",),
```

Comment:
should be ticks/cycle, however cycle does not exist as a unit, so perhaps leave without units